lib/taurus/core/tango/tangodatabase.py

```diff
@@ -679,8 +679,12 @@ def __init__(self, host=None, port=None, parent=None):
 
         # Set host to fqdn
         host = socket.getfqdn(host)
+        try:
+            self.dbObj = Database(host, port)
+        except Exception as e:
+            from taurus import warning
```

Comment:
Since TangoAuthority is a logger object, why not use `self.warning`?

Reply:
Base class is not initialized yet